sdk/cosmos/azure-cosmos-kafka-connect/pom.xml

```diff
@@ -93,6 +104,11 @@ Licensed under the MIT License.
       <scope>test</scope>
       <version>1.10.0</version> <!-- {x-version-update;org.apache.commons:commons-text;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.jayway.jsonpath</groupId>
```

Comment:
I don't think we would want to add any transitive dependencies that aren't whitelisted by the central team here - so, shading or getting rid of this dependency

Reply:
Added shading